plugin/pulp/plugin/importer.py

```diff
@@ -0,0 +1,18 @@
+from pulp.platform.models import RepositoryImporter
+
+
+class RepositoryImporter(RepositoryImporter):
+    """
+    The base RepositoryImporter object
+    """
+
+    def sync(self):
+        """
+        Perform a sync
```

Comment:
Shouldn't that just be the job of the code reviewer to enforce? It doesn't make sense to document in each function that programmers should make sure to follow good coding practices.